### PR TITLE
fix(policymatcher) : fixed audit event alerts for allowed network rules

### PR DESCRIPTION
**Pull request by Prateeknandle**
**Purpose of PR?**:
For AppArmor, audit event alerts are generated for the allowed network rules.
This pr will fix this issue and now alerts will be only generated for actual audit events.

**Does this PR introduce a breaking change?**
no

**If the changes in this PR are manually verified, list down the scenarios covered:**:
tested with policy which allow :
1. tcp protocol 
2. tcp, udp
3. tcp, udp, raw

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [x] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: KubeArmor/feeder/policyMatcher.go [hunk 1]
```diff
@@ -960,7 +973,8 @@ func (fd *Feeder) UpdateMatchedPolicy(log tp.Log) tp.Log {
 		}
```
Comment (daemon1024) [inline KubeArmor/feeder/policyMatcher.go:974]: Can we document what's happening in this particular switch case (`case "Network":`) so as future contributors have context atleast for this part?

Comment (Prateeknandle) [reply KubeArmor/feeder/policyMatcher.go:974]: sure, will add

Comment (Prateeknandle) [reply KubeArmor/feeder/policyMatcher.go:974]: some comments were already added, at specific conditions

**Review by nyrahul — APPROVED** [review]
LGTM

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**